react-native-spokestack.d.ts

```diff
@@ -0,0 +1,97 @@
+export = RNSpokestack;
+export as namespace RNSpokestack;
+
+declare namespace RNSpokestack {
+  export function constructor(props?: object): void;
+
+  export enum TraceLevel {
+    DEBUG = 10,
+    PERF = 20,
+    INFO = 30,
+    NONE = 100,
+  }
+
+  export enum TTSFormat {
+    TEXT = 0,
+    SSML = 1,
+    SPEECHMARKDOWN = 2,
+  }
+
+  interface SpokestackRecognizeEvent {
+    transcript: string;
+  }
+
+  interface SpokestackErrorEvent {
+    error: string;
+  }
+
+  interface SpokestackTraceEvent {
+    message: string;
+  }
+
+  interface SpokestackTTSEvent {
+    url: string;
+  }
+
+  interface NLUSlot {
+    type: string;
+    value: string;
+  }
+
+  export interface SpokestackNLUEvent {
+    result: {
+      intent: string;
+      confidence: string;
+      slots: NLUSlot[];
+    };
+  }
+
+  interface SynthesizeOptions {
+    input: string;
+    format: TTSFormat;
+    voice: string;
+  }
+
+  export interface SpokestackConfig {
+    input: string;
+    stages: string[];
+    properties: {
+      locale?: string;
+      "agc-compression-gain-db"?: number;
+      "google-credentials"?: string; // only set if using `GoogleSpeechRecognizer` stage above
+      "trace-level"?: TraceLevel;
+    };
+    tts: {
+      ttsServiceClass?: string;
+      // These can be generated at spokestack.io after creating an account
+      "spokestack-id"?: string;
+      "spokestack-secret"?: string;
+    };
+    nlu: {
+      // NLU settings. Only set these if you are calling `Spokestack.classify`.
+      "nlu-model-path"?: string; // string filesystem path to nlu model
+      "nlu-metadata-path"?: string; // string filesystem path to nlu metadata
+      "wordpiece-vocab-path"?: string; // string filesystem path to nlu vocab
+    };
```

Comment:
From what I can tell from the README, none of these are actually required.

Reply:
correct